test/SerialIOTest.cpp

```diff
@@ -421,8 +421,7 @@ TEST_CASE("multiple_series_handles_test", "[serial]")
     {
         std::unique_ptr<openPMD::Series> series_ptr;
         {
-            openPMD::Series series(
-                "sample%T.json", openPMD::AccessType::CREATE);
```

Comment:
We left those intentionally to test that the old type still works.

We could silence this at one last location with a local suppressor?

Reply:
I was under the impression that they sneaked in because this test comes from a long-standing pull request (the bug tested by this took a long time to fix).
But we can definitely use the chance to test for existence of AccessType with this